src/styles/index.scss

```diff
@@ -2,2 +1,19 @@
+@use './header.scss';
+@use './card-container';
+
+.page {
+  display: flex;
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+  scroll-behavior: smooth;
+}
```

Comment:
It seems like you're applying global styles to a class `.page`. Usually, such styles are applied to the `body` or `html` tags. If you want these styles to be global, consider moving them to the `body` or `html` tags.